source/ddbus/meson.build

```diff
@@ -0,0 +1,34 @@
+lib_src = [
+    'attributes.d',
+    'bus.d',
+    'c_lib.d',
+    'conv.d',
+    'exception.d',
+    'package.d',
+    'router.d',
+    'simple.d',
+    'thin.d',
+    'util.d',
+]
+
+ddbus_lib = library(
+    'ddbus',
+    lib_src,
+    version: meson.project_version(),
+    install: true,
+    include_directories: src_inc,
+    link_args: '-shared',
```

Comment:
This looks sketchy. Do you really want to just use a `shared_library` here?

Reply:
Oh right. I guess this could be removed anyway since this is only required to work around some meson bug in 0.51.2 or 0.52

Reply:
even if you need to add `-shared` to work around an bug in an old version if someone sets the default library type to both or static meson is going to get grumpy. I'd use `shared_library` and add the `-shared` option to that.

Reply:
Ah, I think I'd just remove the `-shared` and add a warning that building with Meson 0.52 and 0.53 isn't supported, since building with DMD and LDC is broken in those versions anyway, if I changed the `library` to `shared_library` it wouldn't be possible to choose between shared and dynamic libs any more.